src/pages/Controllers/Controllers.js

```diff
@@ -90,15 +91,26 @@ function Details() {
     </span>
   );
 
+  function generatePathValue(c) {
```

Comment:
heh, my bad, I've been doing a lot of typed work lately in which this would have been `function generatePathValue(c: ControllerData) {` :) 

I've updated this.